src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -136,15 +135,6 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
       });
     }
 
-    if (message.isReactable() && !conversation.removed_from_conversation()) {
```

Comment:
Do we still use that property in the codebase? If not, let's remove it from the entity

Reply:
which property?

Reply:
Both isReactable and removed_from_conversation are removed_from_conversation are used in our codebase